contents/handbook/company/management.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Management at PostHog
+sidebar: Handbook
+showTitle: true
+---
+
+As we grow, we'll increase the number of managers at PostHog. Here's what a manager at PostHog looks like.
+
+## Defining the role of manager
+
+A manager at PostHog has these tasks
+1. Making sure their direct reports are happy and productive
+1. Setting the right context for direct reports to do their job
+
+That's it.
+
+A manager at PostHog is _not_ responsible for:
+1. Setting compensation (we have transparent compensation)
+1. Setting tasks for their direct reports
+1. Creating a career path (career paths should be transparent and documented, and for now centrally managed)
+1. "Approving," whether that's projects, expenses, days off or accounts (people should have admin access by default to most things)
+
+
+## What does setting context mean?
+
+At PostHog, we exclusively hire people that are the best in their field.
+That means managers won't need to spend time telling their direct reports what to do.
+
+However, for those people to make the best decisions, they need context. That context can be
+- what a customer said was or wasn't important to them
+- what the metrics are saying needs to be improved
+- what another team in the organisation is working on
+- what the overal goals are for PostHog
```

Comment:
Type: overal**l**

Reply:
Damn - bot didn't catch that one, will add to the list